package.json

```diff
@@ -30,6 +31,7 @@
     ]
   },
   "devDependencies": {
+    "@instacart/tsconfig": "0.1.1",
```

Comment:
This is just an example package I found on npm containing extended configs, for use in the unit tests as fixture, as couldn't mock node modules / require (jest uses require of actual module internally when mocking a require)

Is locked to specific version to keep the tests consistent in future